### PR TITLE
Add module bootstrapann

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] Add bootstrapann to subworkflow
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/tomte _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by Lucpen** [comment]
I added the BootstrapAnn to the allele_specific_calling subworkflow, but perhaps I should just have it on its own as it is looking for the MAE. What do you think?

File: subworkflows/local/allele_specific_calling.nf [hunk 1]
```diff
@@ -43,7 +44,14 @@ workflow ALLELE_SPECIFIC_CALLING {
         )
         ch_versions = ch_versions.mix(GATK4_ASEREADCOUNTER.out.versions.first())
 
+        BOOTSTRAPANN(
+            BCFTOOLS_VIEW.out.vcf.join(BCFTOOLS_INDEX.out.tbi),
```
Comment (jemten) [inline subworkflows/local/allele_specific_calling.nf:48]:
```suggestion
            vcf_tbi,
```

Comment (jemten) [reply subworkflows/local/allele_specific_calling.nf:48]: I think you want to annotate the original vcf. Non-heterozygote variants in the original vcf will simply not get a annotation.

**Review by jemten — COMMENTED** [review]
I think bootstrapann is well placed in the allelle specific wotkflow. Vep should be outside it though.

File: subworkflows/local/allele_specific_calling.nf (hunk 1 above)
Comment (Lucpen) [reply subworkflows/local/allele_specific_calling.nf:48]: You are right, I was still thinking about the asereadcounter

**Review by jemten — APPROVED** [review]
👍

**Pull request merged**